src/components/componentsSearchResults/componentsSearchResults.jsx

```diff
@@ -0,0 +1,30 @@
+// SearchResults.jsx
+import React from 'react';
+import FakeBookings from '@/data/fakeBookings.json';
+import TableHead from './TableHead';
+import TableBody from './TableBody';
+
+const SearchResults = () => {
+  return (
+    <table>
+      <thead>
+        <TableHead />
+      </thead>
+      <tbody>
+        {FakeBookings.map((booking) => (
```

Comment:
The search results should be passed as props to this component. If you can check `<Bookings />` component in `Bookings/Bookings.jsx`, you can see the `FakeBookings` data is passed to `SearchResults` component. See here: https://github.com/RbAvci/React-Module-Project/blob/bbbed68ba6a9d4a0e19be7c916785bf2b0712ab0/src/components/Bookings/Bookings.jsx#L13.